pages/index.js

```diff
@@ -1,16 +1,29 @@
 import styled from "styled-components";
+import useSWR from "swr";
 
-import items from "@/db.json";
 import Item from "@/components/Item";
 
 export default function HomePage() {
+  const { data: items, isLoading, error } = useSWR("/api/items");
+
+  if (error) {
+    console.log(error.message);
+    return <h2>{error.message}</h2>;
```

Comment:
If you return the error.message in your h2 element then the console.log is not needed or? As console.logs are part of the development they should not be part of the finished feature. :)